stl/inc/istream

```diff
@@ -649,8 +654,12 @@ public:
         _Myios::clear(_Oldstate & ~ios_base::eofbit);

```

Comment:
I won't demand that you do it - since it's not even visible without expanding the context around the change - but I wouldn't complain if you wrap the comment on 651 so the function declaration is all on one line.

Reply:
I went through all of `ostream` and `istream` and fixed all wrap inducing comments